content/docs/user-guide/components/reference/physx/rigid-body.md

```diff
@@ -0,0 +1,54 @@
+
+Dynamic rigid bodies are fully simulated by PhysX. Dynamic rigid bodies move in response to collision events and forces and are not animated through motions or scripts. By default, dynamic rigid bodies are affected by gravity, but gravity can be deactivated in the PhysX Rigid Body component.
+
+Kinematic rigid bodies are not fully simulated by PhysX. Kinematic rigid bodies have scripted animation and are not affected by forces or gravity. Doors, for example, often have scripted animation. If the door is a kinematic rigid body, the door can move through scripted animation and collide with other PhysX entities during simulation. Movement is created with the `SetKinematicTarget` method that you specify in a script.
+
+{{< note >}}
+You should always add the PhysX Rigid Body component to the top level of an entity hierarchy. Adding the PhysX Rigid Body component to a child entity can cause conflicts with the entity's world transform and result in undefined behavior.
+{{< /note >}}
+
+## Provider
+
+[PhysX Gem](/docs/user-guide/gems/reference/physx/)
+
+## Properties 
+
+![PhysX Rigid Body component interface.](/images/user-guide/components/reference/physx/physx-rigid-body-ui-01.png)
+
+| Property | Description | Value | Default |
+| - | - | - | - |
+| **Initial linear velocity** | Sets the starting linear velocity of the rigid body when the entity is activated. This creates movement in the direction of the linear velocity vector. | Vector3: -Infinity to Infinity | X: `0.0`, Y: `0.0`, Z: `0.0` |
+| **Initial angular velocity** | Sets the starting angular velocity of the rigid body when the entity is activated. This creates rotation in the direction of the angular velocity vector. | Vector3: -Infinity to Infinity | X: `0.0`, Y: `0.0`, Z: `0.0` |
+| **Linear damping** | Sets the rate of decay over time for linear velocity even if no forces are acting on the rigid body. A non-zero value eventually stops the rigid body if no linear force is applied. Value must be non-negative. | Float: 0 to Infinity | `0.05` |
+| **Angular damping** | Sets the rate of decay over time for angular velocity even if no forces are acting on the rigid body. A non-zero value eventually stops the rigid body if no torque force is applied. Value must be non-negative. | Float: 0 to Infinity | `0.15` |
+| **Sleep threshold** | Sets the kinetic energy per unit mass below which the rigid body can go to sleep. Value must be non-negative. | Float: 0 to Infinity | `0.005` |
+| **Start asleep** | If enabled, the Rigid Body component is asleep when the entity is activated and wakes when a sufficient force is applied. | Boolean | `Disabled` |
```

Comment:
```suggestion
| **Start asleep** | If enabled, the PhysX Rigid Body component is asleep when the entity is activated and wakes when a sufficient force is applied. | Boolean | `Disabled` |
```

Reply:
@chanmosq I don't think you meant to include this edit.